wordpress/1.11/README.md

```diff
@@ -0,0 +1,239 @@
+# How to use WordPress on DC/OS
+
+[WordPress](https://wordpress.com/) is open source content management system based on PHP and MySQL that you can use to create a beautiful website, blog, or app.
+
+- Estimated time for completion: 10 minutes
+- Target audience:
+    - Bloggers
+    - Developers
+- Scope: Learn how to deploy WordPress on DC/OS.
+
+
+**Table of Contents**:
+
+- [Prerequisites](#prerequisites)
+- [Install MySQL using GUI](#install-mysql-using-gui)
+- [Install MySQL using CLI](#install-mysql-using-cli)
+- [Customize MySQL](#customize-mysql)
+- [Install WordPress using GUI](#install-wordpress-using-gui)
+- [Install WordPress using CLI](#install-wordpress-using-cli)
+- [Access WordPress](#access-wordpress)
+- [Use WordPress in production](#use-wordpress-in-production)
+
+
+## Prerequisites
+
+- DC/OS 1.11 or later
+- [Marathon-LB](https://docs.mesosphere.com/services/marathon-lb/) must be installed and running
+- Identify [IP of the public agent](https://docs.mesosphere.com/1.11/administering-clusters/locate-public-agent/) where Marathon-LB or an available hostname configured to point to the public agent(s) where Marathon-LB is running.
+- DC/OS [CLI Setup](https://docs.mesosphere.com/1.11/cli/install)
+
+## Install MySQL using GUI
+
+- Login to DC/OS admin, click on Catalog on the left menu bar, search `mysql` and select the package.
+
+![MySQL Search](img/mysql-search.png)
+
+- Go to Database tab and change config as below:
+  - name: wordpress
+  - username: wp_admin
+  - password: password
+
+![MySQL Config Database](img/mysql-config-database.png)
+
+- Go to Networking tab and enable(check) host mode
+
+![MySQL Config Networking](img/mysql-config-networking.png)
+
+- Finally, click on `Review & Run` button.
+
+![MySQL Install](img/mysql-install.png)
+
+- Wait for service to be up and running, go to Services section and check the service status.
+- Now, go to [Customize MySQL](#customize-mysql) section.
+
+## Install MySQL using CLI
+
+Create a file called `mysql-config.json` and set the following properties:
+
+```json
+{
+  "service": {
+    "name": "mysql"
+  },
+  "database": {
+    "name": "wordpress",
+    "username": "wp_admin",
+    "password": "password",
+    "root_password": "wordpress"
```

Comment:
Just noting that this does differ slightly from the screenshot above, where the root password was unchanged.

Reply:
sorry, my mistake. I was doing some experiment and forgot to change back.